src/internal/svg/fundWallet.tsx

```diff
@@ -0,0 +1,19 @@
+import { fill } from '../../styles/theme';
+
+export const FundWalletSvg = (
+  <svg
+    role="img"
+    aria-label="fund-wallet-svg"
+    width="16"
+    height="16"
+    viewBox="0 0 16 16"
```

Comment:
```suggestion
    width="100%"
    height="100%"
    viewBox="0 0 20 20"
```

Reply:
This should be more flexible as it allows the parent div to control the sizing

Reply:
done, thanks!